### PR TITLE
updated nrt_observation_coded query

### PR DESCRIPTION
**Pull request by ndduc01**
## Notes

Please include a summary of the change and which issue is fixed or which feature is added. 

## JIRA

- **Related story**: [Jira Ticket](url)

## Checklist

- [ ] PR focuses on a single story.
- [ ] New unit tests added and ensured they pass.
- [ ] Service has been tested in local and it works as expected.
- [ ] Documentation has been updated for this code change (if needed).
- [ ] Code follows the Java Coding Conventions (https://www.oracle.com/java/technologies/javase/codeconventions-programmingpractices.html).

## Types of changes

What types of changes does this PR introduces?

- [ ] Bugfix
- [ ] New feature
- [ ] Breaking change

## Testing

- [ ] Does this PR has >90% code coverage?
- [ ] Is the screenshot attached for code coverage?
- [ ] Does the `gradle build` pass in your local? 
- [ ] Is the `gradle build` logs attached?

### CODE REVIEW

File: liquibase-service/src/main/resources/db/rdb/routines/001-generate_data_for_data_sync_config-004.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
+DELETE
+FROM [dbo].[data_sync_config]
+WHERE table_name = 'NRT_OBSERVATION_CODED';
+
+
+INSERT INTO [dbo].[data_sync_config]
```
Comment (upasanapattnaik-eq) [inline liquibase-service/src/main/resources/db/rdb/routines/001-generate_data_for_data_sync_config-004.sql:6]: 👍

**Review by upasanapattnaik-eq — APPROVED** [review]
LGTM!

File: liquibase-service/src/main/resources/db/rdb/routines/001-generate_data_for_data_sync_config-004.sql [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        FROM rdb_modern.dbo.nrt_observation_coded
+        JOIN rdb_modern.dbo.nrt_observation
+        ON rdb_modern.dbo.nrt_observation_coded.observation_uid = rdb_modern.dbo.nrt_observation.observation_uid
+        WHERE rdb_modern.dbo.nrt_observation_coded.refresh_datetime :operator :timestamp
+        OR rdb_modern.dbo.nrt_observation.last_chg_time :operator :timestamp
+        OR rdb_modern.dbo.nrt_observation.refresh_datetime :operator :timestamp
+    )
+    SELECT * FROM PaginatedResults
+    WHERE RowNum BETWEEN :startRow AND :endRow;')
+;
```
Comment (alog-enquizit) [inline liquibase-service/src/main/resources/db/rdb/routines/001-generate_data_for_data_sync_config-004.sql:33]: nitpick: can this semicolon be in the previous statement :)

File: liquibase-service/src/main/resources/db/rdb/routines/001-generate_data_for_data_sync_config-004.sql (hunk 1 above)
Comment (alog-enquizit) [inline liquibase-service/src/main/resources/db/rdb/routines/001-generate_data_for_data_sync_config-004.sql:6]: If possible add a few lines of comment on the scope of this script.

**Review by alog-enquizit — APPROVED** [review]
LGTM 👍

File: liquibase-service/src/main/resources/db/rdb/routines/001-generate_data_for_data_sync_config-004.sql (hunk 2 above)
Comment (ndduc01) [reply liquibase-service/src/main/resources/db/rdb/routines/001-generate_data_for_data_sync_config-004.sql:33]: yes, that for the service to handle. No issue

**Pull request merged**